offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/GridUtils.groovy

```diff
@@ -96,4 +134,12 @@ class GridUtils {
         }
     }
 
+    private static void styleFilterTextField(TextField filterTextField, String columnCaption) {
+        filterTextField.setValueChangeMode(ValueChangeMode.EAGER)
+        filterTextField.addStyleName(ValoTheme.TEXTFIELD_TINY)
+        filterTextField.setPlaceholder("Filter by $columnCaption")
+        filterTextField.setSizeFull()
+
+    }
+
```

Comment:
imo you could use that method also in the other filter method

Reply:
you are right :D totally forgot about that